BoldGrid/crio
# Add additional feat-image controls for post headers

Resolves #9

[crio-2.19.0-issue-9.zip](https://github.com/BoldGrid/crio/files/10521410/crio-2.19.0-issue-9.zip)

Testing Instructions
1. Install Crio and activate starter content ( or install an inspiration with a blog, then replace with this zip ).
2. Open the Customizer and navigate to `Design > Blog > Posts > Featured Images`
3. Ensure that you see four options for the `Position` control. They should be `Header Background, Above Header, Below Header, and In Content`
4. Select all four options, ensuring they appear correctly in the preview, publish, and check that they appear correctly on the front end.
5. When selecting `Above Header and Below Header` you will see another control appear called `Featured Image Display Type` which will allow you to select either `Background` or `Image` . Ensure both options display correctly in both positions, in the preview and when published.
6. When choosing the type `background` you will see a slider show up allowing you to specify the height of the background. Ensure the control changes are reflected in the preview and when published, for both above and below header.
7. When choosing the type `image` you can select the image size and alignment. Ensure these controls are working in the preview and when published.
8. When choosing the position `In Content` ensure that the image size and alignment controls are shown, and work as expected.
